sdk/identity/identity/Troubleshooting.md

```diff
@@ -203,46 +266,50 @@ The supported clouds are:
+|Az.Account module >= 2.2.0 isn’t installed.|The Az.Account module needed for authentication in Azure PowerShell isn’t installed.|Install the latest Az.Account module. Installation instructions can be found [here](https://docs.microsoft.com/powershell/azure/install-az-ps). To use the Azure PowerShell credential, install the Azure Az PowerShell module:```powershell
 Install-Module -Name Az -Scope CurrentUser -Repository PSGallery -Force
-```
+```|
+|Please run 'Connect-AzAccount' to set up account.|No account is currently logged into Azure PowerShell.|<ul><li>Login to Azure PowerShell using the `Connect-AzAccount` command. More instructions for authenticating Azure PowerShell can be found [here](https://docs.microsoft.com/powershell/azure/authenticate-azureps)</li><li>Validate that Azure PowerShell can obtain tokens. See [below](#verifying-azure-powershell-can-obtain-tokens) for instructions.</li></ul>|
 
-Then try running the credential again.
+#### __Verifying Azure PowerShell can obtain tokens__
+You can manually verify that Azure PowerShell is properly authenticated, and can obtain tokens. First use the `Get-AzContext` command to verify the account which is currently logged in to the Azure CLI. 
 
-#### Azure account not logged in
-
-The Azure PowerShell credential utilizes the currently logged in Azure user in Azure PowerShell to fetch an access token.
-You need to log in to your account in Azure PowerShell via the `Connect-AzAccount` command. For further instructions, see [Sign in with Azure PowerShell](https://docs.microsoft.com/powershell/azure/authenticate-azureps?view=azps-6.3.0). Once logged in, try running the credential again.
-
-#### Deserialization error
-
-The Azure PowerShell credential was able to retrieve a response when attempting to get an access token but failed to parse that response. In your local PowerShell window, run the following command to ensure that Azure PowerShell returns an access token in the correct format:
-
-```powershell
-Get-AzAccessToken -ResourceUrl "<Scopes-Url>"
 ```
+PS C:\> Get-AzContext
+Name                                     Account             SubscriptionName    Environment         TenantId
+----                                     -------             ----------------    -----------         --------
+Subscription1 (xxxxxxxx-xxxx-xxxx-xxx... test@outlook.com    Subscription1       AzureCloud          xxxxxxxx-x...
+```
+Once you've verified Azure PowerShell is using correct account, you can validate that it’s able to obtain tokens for this account.
 
+```bash
+Get-AzAccessToken -ResourceUrl "https://management.core.windows.net"
+```
 If the preceding command isn't working properly, follow the instructions to resolve the Azure PowerShell issue. Then try running the credential again.
 
+>Note that output of this command will contain a valid access token, and SHOULD NOT BE SHARED to avoid compromising account security.
+
```

Comment:
Can we add a separator here? like `---`

---

Like that. What do you think?

Reply:
You mean before the note or after?
I don't mind - but we haven't added a separator anywhere else in the file - if we want to add - let's standardize that across sections.  After what sections do we want to add separators?